plugins/builder/traditional_builder.py

```diff
@@ -97,28 +99,26 @@ def commands(self):
 
         # handle any options
         if texify or latexmk:
-            # we can only handle aux_directory, output_directory, or jobname
-            # with latexmk
+            # aux_directory, output_directory or jobname are only supported by latexmk
             if latexmk:
-                if self.aux_directory is not None and self.aux_directory != self.output_directory:
-                    # DO NOT ADD QUOTES HERE
-                    cmd.append("--aux-directory=" + self.aux_directory)
-
-                if self.output_directory is not None:
-                    # DO NOT ADD QUOTES HERE
-                    cmd.append("--output-directory=" + self.output_directory)
+                if self.aux_directory:
+                    # Don't use --aux-directory as the way how latexmk moves
+                    # final documents to possibly defined --output-directory
+                    # prevent file-reloading in SumatraPDF or even fail at all,
+                    # if document is opened and locked by viewer on Windows.
```

Comment:
I don't understand the comment from 105 to 108.

Is this close to what you mean?

```
                    # Don't use --aux-directory as the way latexmk moves
                    # final documents to a possibly defined --output-directory
                    # prevents files reloading in SumatraPDF or even fails
                    # if documents are opened and locked by viewer on Windows.
```

Reply:
I should've known, you are watching :-D

Yes it is.